services/GovDeliveryService.ts

```diff
@@ -173,6 +228,29 @@ export default class GovDeliveryService implements MonitoredService {
     return this.sendEmail(email);
   }
 
+  public sendProductionAccessEmail(supportRequest: ProductionAccessSupportEmail): Promise<EmailResponse> {
+    const email: EmailRequest = {
+      subject: 'Production Access Requested',
```

Comment:
might be helpful to include the organization name in the subject – guessing that'll make it easier to sort through these in the inbox

Reply:
added in latest commit